semversioner/core.py

```diff
@@ -101,6 +101,17 @@ def release(self) -> Release:
 
         return Release(version=next_version_number, changes=changes)
 
+    def next_version(self) -> str:
```

Comment:
There is no need to create a new method called `next_version` as there is already a method to retrieve the next version called `get_next_version`. Let's use that method to reduce duplication.

Reply:
My thoughts was that get_next_version takes a changeset and current_version_numbers as argument. If the method would get those itself, but that would lead to double disk access when releasing.

I suppose get_next_version could accept null/None values, and if is that, read the info from disk?

Reply:
Since you are not worried about the extra FS calls, I've changed my PR to simply getting the information directly in-method.